clients/java/src/main/java/io/zeebe/client/impl/ZeebeCallCredentials.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.zeebe.client.impl;
+
+import io.grpc.Metadata;
+import io.grpc.SecurityLevel;
+import io.zeebe.client.CredentialsProvider;
+import java.util.concurrent.Executor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ZeebeCallCredentials extends io.grpc.CallCredentials {
+  private static final Logger LOG = LoggerFactory.getLogger("io.zeebe.client.impl");
```

Comment:
Can we use a `ZbLogger`? Not that they're much different, but since we typically use this...